src/install.rs

```diff
@@ -241,6 +241,27 @@ pub fn install(config: &InstallConfig) -> Result<()> {
         bail!("install failed");
     }
 
+    match get_filesystems_with_label("boot") {
+        Ok(pts) => {
+            let rootdev = std::fs::canonicalize(&config.device)
+                .unwrap_or_else(|_| PathBuf::from(&config.device))
+                .to_string_lossy()
+                .to_string();
+            let pts = pts
+                .iter()
+                .filter(|pt| !pt.contains(&rootdev))
+                .collect::<Vec<_>>();
+            if !pts.is_empty() {
+                eprintln!("Note: detected other devices with a filesystem labeled 'boot'");
+                for pt in pts {
+                    eprintln!("- {}", pt);
+                }
+                eprintln!("Before booting the freshly installed system, consider wiping these devices with `wipefs -a`");
```

Comment:
I wonder if we can add some more context here.  "Found some other partitions; consider destroying them" seems like dangerous advice on its own.

Reply:
any ideas what to print instead?

Reply:
Maybe something like "The installed OS may not work correctly if there are multiple boot filesystems.  Before rebooting, investigate whether these filesystems are needed and consider wiping them with `wipefs -a`."?

If we get this completely robust (e.g. with the followup mentioned in https://github.com/coreos/coreos-installer/pull/671#issue-1039979287), so that the warning is no longer valid, I assume we'll just remove the check.